src/lib/select/select.html

```diff
@@ -1,6 +1,11 @@
 <div class="md-select-trigger" cdk-overlay-origin (click)="toggle()" #origin="cdkOverlayOrigin" #trigger>
-  <span class="md-select-placeholder" [class.md-floating-placeholder]="this.selected"
-   [@transformPlaceholder]="_placeholderState" [style.width.px]="_selectedValueWidth"> {{ placeholder }} </span>
+  <span
+    class="md-select-placeholder"
+    [class.md-floating-placeholder]="selected"
+    [@transformPlaceholder]="placeholderState"
+    [style.visibility]="(floatingPlaceholder !== 'never' || !selected) ? 'visible' : 'hidden'"
```

Comment:
Can you pull this expression into the class? It's a bit long to be in the template.